test/validators.js

```diff
@@ -10680,6 +10680,38 @@ describe('Validators', () => {
         'FS AB 1234 A',
       ],
     });
+    test({
+      validator: 'isLicensePlate',
+      args: ['it-IT'],
+      valid: [
+        'AA 000 AA',
```

Comment:
For sure `000` is never used on plates in real life :thinking:

Reply:
It was actually registered in the [province of Terni](https://it.wikipedia.org/wiki/Targhe_d%27immatricolazione_dell%27Italia#1994%E2%80%931999)

Reply:
:fearful: